lib/tasks/schools_from_yml.rake

```diff
@@ -0,0 +1,28 @@
+namespace :schools do
+  desc "Reads from the ./config/code_schools.yml and creates records in the database."
+  task populate: :environment do 
+    schools =  YAML::load_file(File.join("./config", "code_schools.yml"), "r")    
+    schools.each do |school|
+      object = CodeSchool.create!(
```

Comment:
Can you add in a `rescue` block so that we catch any errors, and the rest of the schools that are error free can be created.